xyzprinting_metallic_pla.xml.fdm_material

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
+      <material>Premium Metallic PLA</material>
```

Comment:
The material tag is about the material _type_. So the type of this material is PLA, not Premium Metallic PLA.

Reply:
OK~thank you :)